src/rust/ide/src/controller/graph/executed.rs

```diff
@@ -171,6 +164,32 @@ impl Handle {
         Ok(())
     }
 
+    /// Attempts to get the method pointer of the specified node.
+    ///
+    /// Fails if there's no information about target method pointer (e.g. because node value hasn't
+    /// been yet computed by the engine) or if method graph cannot be created (see
+    /// `graph_for_method` documentation).
```

Comment:
outdated doc, this method gets method pointer information as a parameter

Reply:
@mwu-tow I don't get it? This method returns method pointer, doesn't get.

Reply:
@farmaazon Right. The issue stands but my comment was about the wrong part of the docstring:
this function can fail if there is not information about target method pointer (this is correct)
but it doesn't do anything related to the `graph_for_method`